curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63b6152e6aff882db819fc1e.md

```diff
@@ -7,10 +7,12 @@ dashedName: step-18
 
 # --description--
 
-Even though you set an `input` element to be a number, JavaScript receives a string value. You need to write a function to clean the string value and ensure you have a number.
+When the user provides a daily calorie budget, the input will only accept numbers. However, if the user provides a number with a `+` or `-` sign, then you will need to filter those characters out.
 
 Start by declaring a `cleanInputString` function that takes a `str` parameter.
 
+*NOTE*: JavaScript will receive the input as a string, so you will need to convert the string to a number before performing any calculations. You will learn how to convert string values into a number in a future step. 
```

Comment:
I don't know if we should say JavaScript input field because it would be an HTML input.

Maybe this instead?

```md
*NOTE*: Values from an HTML `input` field will be received as strings in JavaScript. You will need to convert the string to a number before performing any calculations. You will learn how to convert string values into a number in a future step. 
```